src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx

```diff
@@ -13,6 +22,17 @@ const SettingsEditProfileRoute: React.FC<SettingsEditProfileRouteProps> = ({
 }) => {
   return (
     <GridColumns>
+      <Column span={8}>
```

Comment:
Both the box and the flex aren't needed. Set display flex on the column, remove the mb as well.

Reply:
Notice also instance how that icon shrunk.

Reply:
👁️